dashboard/test/ui/runner.rb

```diff
@@ -415,8 +415,11 @@ def report_tests_finished(start_time, run_results)
   end
 
   extra_dimensions = {test_type: test_type}
+  total_tests = suite_success_count + failures.count
```

Comment:
Is there a particular reason to calculate this manually rather than just using `run_results.count`?

Reply:
Updated!